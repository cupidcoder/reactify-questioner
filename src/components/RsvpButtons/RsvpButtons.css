.RsvpButtons {
  display: flex;
  width: 90%;
  justify-content: space-between;
  margin: 0.5rem auto;
  border-bottom: 1px solid rgba(46,62,72,.12);
}

.RsvpText h3 {
  font-size: 1.4rem;
  margin-top: 0.3rem;
  margin-left: 1rem;
  font-style: italic;
}

.RsvpResponse {
  display: flex;
}

.RsvpResponse button:not(:last-child) {
  margin-right: 0.7rem;
}
