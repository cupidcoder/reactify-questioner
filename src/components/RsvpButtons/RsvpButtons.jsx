import React from 'react';
import Button from '../Button/Button';
import style from './RsvpButtons.css';

const RsvpButtons = () => (
  <div className={style.RsvpButtons}>
    <div className={style.RsvpText}>
      <h3>Are you going?</h3>
    </div>
    <div className={style.RsvpResponse}>
      <Button>Yes</Button>
      <Button>No</Button>
      <Button>Maybe</Button>
    </div>
  </div>
);

export default RsvpButtons;
