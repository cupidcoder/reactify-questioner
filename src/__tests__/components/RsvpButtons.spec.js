import React from 'react';
import { shallow } from 'enzyme';
import RsvpButtons from '../../components/RsvpButtons/RsvpButtons';

describe('RsvpButtons', () => {
  it('Should correctly save snapshot of the RsvpButtons', () => {
    const wrapper = shallow(<RsvpButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});
