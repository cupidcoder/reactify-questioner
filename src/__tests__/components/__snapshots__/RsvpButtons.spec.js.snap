// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RsvpButtons Should correctly save snapshot of the RsvpButtons 1`] = `ShallowWrapper {}`;
